src/aiida/orm/utils/builders/code.py

```diff
@@ -15,14 +15,16 @@
 from aiida.common.warnings import warn_deprecation
 from aiida.orm import InstalledCode, PortableCode
 
-warn_deprecation('This module is deprecated. To create a new code instance, simply use the constructor.', version=3)
```

Comment:
Moved this inside the class constructor, otherwise it's very hard to catch this warning